### PR TITLE
Add additional nationalAccountId field to enable national bank transfers in Argentina

### LINKED ISSUE #1489: Required data fields for sending money in Argentina
I tried to 'Create new account' with the payment methods 'National bank transfer', 'Transfer with same bank' and 'Transfers with specific banks' without success. The issue is that in Argentina we identify bank accounts with a CBU number instead of IBAN, BIC or SWIFT.

I'll provide the required data fields for sending money (English and Spanish version), but first I would like to talk about how the CBU number is created because it may be used to fill some fields automatically and detect if the CBU is invalid.

CBU is a 22 numerical digits that looks like 2850590940090418135201 (example used in [1]).

Field explanation:

If the CBU is aaabbbbcddddddddddddde, then:
aaa = Bank code. Using the table in [1] then you know the Bank name (Nombre del banco)
bbbb = Branch no. (Número de sucursal)
c = checksum of aaa and bbbb
ddddddddddddd = Account no. (Número de cuenta)
e = checksum of ddddddddddddd

The checksum is explain in [1]. I can translate it for you if you would like to implemented it.

Now, the required data fields for sending money using 'National bank transfer', 'Transfer with same bank' and 'Transfers with specific banks':

- **Account owner full name** (**Nombre del propietario de la cuenta**)
- **Account owner CUIL/CUIT** (**CUIL/CUIT del propietario de la cuenta**): 11 numerical digits usually written in this way: xx-xxxxxxxx-x (you can see more details in [2]).
- **CBU no.** (**Número de CBU**): 22 numerical digits
- **Bank name** (**Nombre del banco**) (*)
- **Branch no.** (**Número de sucursal**) (*)
- **Account no.** (**Número de cuenta**) (*)

(*) Could be automatically filled using the previous explanation. Manually filled is ok.

[1] https://es.wikipedia.org/wiki/Clave_Bancaria_Uniforme#Estructura_de_la_CBU
[2] https://es.wikipedia.org/wiki/Clave_Única_de_Identificación_Tributaria

@flix1 fyi

### PR DESCRIPTION
**Pull request by ripcurlx**
Fixes #1489.

### CODE REVIEW

**Review by ManfredKarrer — APPROVED** [review]
utACK

File: src/main/java/bisq/desktop/components/paymentmethods/SameBankForm.java [hunk 1]
```diff
@@ -51,37 +51,4 @@ public SameBankForm(PaymentAccount paymentAccount, AccountAgeWitnessService acco
...
-    }
-
-    @Override
-    protected void addHolderNameAndIdForDisplayAccount() {
-        Tuple2<Label, TextField> tuple = addLabelTextField(gridPane, ++gridRow, Res.getWithCol("payment.account.owner"));
-        TextField holderNameTextField = tuple.second;
-        holderNameTextField.setMinWidth(300);
-        holderNameTextField.setText(bankAccountPayload.getHolderName());
-    }
-
```
Comment (ripcurlx) [inline src/main/java/bisq/desktop/components/paymentmethods/SameBankForm.java:97]: @ManfredKarrer Is there a reason for these overrides in the Same Bank Form case?

Comment (ManfredKarrer) [reply src/main/java/bisq/desktop/components/paymentmethods/SameBankForm.java:97]: Hm.. don't remember why it was overridden. Just test it to see if all works as expected.

Comment (ripcurlx) [reply src/main/java/bisq/desktop/components/paymentmethods/SameBankForm.java:97]: Worked so far for me.

**Pull request merged**